apps/ledger-live-mobile/src/screens/ReceiveFunds/01-SelectCrypto.tsx

```diff
@@ -64,7 +69,7 @@ export default function AddAccountsSelectCrypto({ navigation, route }: Props) {
   );
   const cryptoCurrencies = useMemo(
     () =>
-      listSupportedCurrencies()
+      (listSupportedCurrencies() as (CryptoCurrency | TokenCurrency)[])
```

Comment:
Maybe use `CryptoOrTokenCurrency ` ?

Reply:
I did not mention all the files that have this possible improvement

Reply:
Yes, well, it's really a huge issue to use the union I would say

Reply:
Yeah of course :)